src/tqdm_publisher/publisher.py

```diff
@@ -9,22 +9,95 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.callbacks = {}
 
-    # Override the update method to call callbacks 
-    def update(self, n=1, always_callback=False):
-        if super().update(n) or always_callback:
 
+    # Override the update method to call callbacks 
+    def update(self, n=1):
+        if super().update(n):
```

Comment:
The `if` here is a tad confusing on first glance; can you annotate what the return types expected of this function are?

Reply:
This returns [True or None based on whether a `display()` was triggered](https://tqdm.github.io/docs/tqdm/#update). Though we actually might not care about this and should let the client decide?

Reply:
Thanks for the link; we should _probably_ mimic or at least pass whatever the behavior is so that our injection provides minimal disruption form expectation

Therefore I would capture the result of the call to the `super` and return it after the injection

Reply:
Ah got it